tests/test_scavenging.py

```diff
@@ -541,3 +545,23 @@ def test_multiple_definition(v):
     check(v.defined_vars, ['a', 'a'])
     check(v.used_names, [])
     check(v.unused_vars, ['a', 'a'])
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6),
+                    reason="requires python3.6 or higher")
+def test_typing_module(v):
+    v.scan("""\
+from typing import Iterable, List
+
+x: List[int] = [1]
+
+def f(n: int) -> Iterable[int]:
+    i = 0
+    while i < n:
+        yield i
+        i += 1
+""")
+
+    check(v.unused_vars, ['x'])
+    check(v.unused_funcs, ['f'])
+    check(v.used_names, ['Iterable', 'n', 'i', 'int', 'List'])
```

Comment:
I think it would be a good idea to add the following case as well (perhaps in a different test function):

```py
import typing

if typing.TYPE_CHECKING:
    from typing import List, Text

def foo(foo_li):
    # type: (List[Text]) -> None
    for bar in foo_li:
        bar.xyz()
```

In this snippet, you should assert that List, Text indeed are being rendered unused - just to make a point that Vulture does not support parsing of comments.

Reply:
Sure thing. The test has been added